chomp-frontend/src/layouts/AuthLayout.tsx

```diff
@@ -0,0 +1,8 @@
+import { Footer } from "../features/footer/Footer";
+
+export const AuthLayout = ({ children }: { children: React.ReactNode }) => (
+  <div className="flex flex-col min-h-screen">
+    <main className="flex-grow">{children}</main>
```

Comment:
im not 100% sure if this would work but I know there is an Outlet element for specifying where children are rendered

```suggestion
    <Outlet className="flex-grow" />
```